nauplius.csl

```diff
@@ -0,0 +1,489 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Nauplius</title>
+    <id>http://www.zotero.org/styles/nauplius</id>
+    <link href="http://www.zotero.org/styles/nauplius" rel="self"/>
+    <link href="http://www.zotero.org/styles/water-alternatives" rel="template"/>
+    <link href="http://www.scielo.br/revistas/nau/iinstruc.htm" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <eissn>2358-2936</eissn>
+    <updated>2019-10-28T14:52:15+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-US">
+    <date form="numeric">
+      <date-part name="day" suffix=" " range-delimiter="-"/>
+      <date-part name="month" suffix=" "/>
+      <date-part name="year" range-delimiter="/"/>
+    </date>
+  </locale>
+  <macro name="container">
+    <choose>
+      <if type="paper-conference speech" match="any">
+        <choose>
+          <if variable="container-title event" match="any">
+            <text value="In "/>
+          </if>
+        </choose>
+        <group delimiter=", ">
+          <names variable="editor translator" delimiter=", ">
+            <name name-as-sort-order="all" sort-separator=", " initialize-with="." delimiter="; " and="text" delimiter-precedes-last="never"/>
+            <label form="short" prefix=" (" suffix=")"/>
+          </names>
+          <group delimiter=" ">
+            <choose>
+              <if variable="ISBN ISSN container-title" match="any">
+                <group font-style="italic">
+                  <choose>
+                    <if variable="container-title">
+                      <text variable="container-title"/>
+                    </if>
+                    <else>
+                      <text variable="event"/>
+                    </else>
+                  </choose>
+                  <text variable="genre"/>
+                  <text variable="collection-title"/>
+                </group>
+              </if>
+              <else>
+                <choose>
+                  <if variable="container-title">
+                    <text variable="container-title"/>
+                  </if>
+                  <else>
+                    <text variable="event"/>
+                  </else>
+                </choose>
+                <text variable="genre"/>
+                <text variable="collection-title"/>
+              </else>
+            </choose>
+          </group>
+        </group>
+      </if>
+      <else-if type="chapter" match="any">
+        <group delimiter=", ">
+          <choose>
+            <if variable="ISBN ISBN" match="any">
```

Comment:
I don't understand the test here -- don't see why a book chapter with an ISBN would be cited differently?